apps/checkout-app/mocks/consts.ts

```diff
@@ -0,0 +1,23 @@
+export type TestingEnvVar =
```

Comment:
Do we need type? You can also create an object and then `export type TestingEnvVar = keyof<typeof testingVars>`

But this is just a suggestion and probably both are fine